### PR TITLE
Add Login process

### PR DESCRIPTION
**Pull request by aytouna**

### CODE REVIEW

File: src/main/java/net/dcess/gestrec/boundary/LoginRequest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import javax.validation.constraints.NotEmpty;
+import lombok.Data;
+
+/**
+ *
+ * @author aaytouna
+ */
+@Data
+public class LoginRequest implements Serializable {
+    @Column(nullable = false)
```
Comment (JalalSordo) [inline src/main/java/net/dcess/gestrec/boundary/LoginRequest.java]: why columnthis is not a table in the database...its just a dto

File: src/main/java/net/dcess/gestrec/entity/Account.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package net.dcess.gestrec.entity;
+
+/**
+ *
+ * @author aaytouna
+ */
+public class Account {
+
```
Comment (JalalSordo) [inline src/main/java/net/dcess/gestrec/entity/Account.java:29]: account is empty ? how did you test ?

**Review by JalalSordo — CHANGES_REQUESTED** [review]
rework check comments

File: src/main/java/net/dcess/gestrec/entity/Account.java (hunk 2 above)
Comment (aytouna) [reply src/main/java/net/dcess/gestrec/entity/Account.java:29]: i m waiting for the account class to be created

**Review by JalalSordo — APPROVED** [review]
merged

**Pull request merged**